src/Redux/Rockets/RocketsSlice.js

```diff
@@ -0,0 +1,54 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const GET = 'space-travelers-hub/rocketsReducer/GET';
+const BOOK = 'space-travelers-hub/rocketsReducer/BOOK';
+
+// Book rocket
+const reserveRocket = (state, payload) => {
+  const newstate = state.map((rocket) => {
+    if (rocket.id !== payload) return rocket;
+    return { ...rocket, reserved: !rocket.reserved };
+  });
+  return newstate;
+};
```

Comment:
- [ ] I think it would be better if you handle booking and leaving actions separately, they modify the same property of a rocket, but they are different actions. When I see the logger, and I leave a mission, I still see the `'space-travelers-hub/rocketsReducer/BOOK'` action, which doesn't make much sense. Please consider applying this! 🤓 🔧